appimage/generate-appimages.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+function get_tools_and_plugins() {
+  wget https://github.com/linuxdeploy/linuxdeploy/releases/download/continuous/linuxdeploy-x86_64.AppImage
```

Comment:
If a user wants to build locally like you mentioned in the description and updates their repo, would it be worth checking for the existence of this file? Otherwise I believe `wget` will download a second copy.